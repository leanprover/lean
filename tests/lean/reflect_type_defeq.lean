meta def foo (x : reflected (3 : ℕ)) : reflected ([10] : list ℕ) :=
x -- ERROR