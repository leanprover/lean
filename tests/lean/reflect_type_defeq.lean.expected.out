reflect_type_defeq.lean:2:0: error: equation type mismatch, expression
  x
has type
  reflected 3
but is expected to have type
  reflected [10]
